locales/en-US/notes.properties

```diff
@@ -107,3 +107,6 @@ textDirectionTitle=Text Direction
 themeLegend=Theme
 defaultThemeTitle=Default
 darkThemeTitle=Dark
+
+# Context Menu option text
```

Comment:
Suggested comment:

```
# Displayed in the context menu when selecting text on a web page, used to send
# the text to Firefox Notes. Keep "Notes" in English, since it's the name of the
# add-on.
```